src/roswire/ros2/node.py

```diff
@@ -2,28 +2,57 @@
 __all__ = ('ROS2Node',)
 
 import typing
-
 import attr
```

Comment:
Should appear in same block of imports as `dockerblade`. The `from` import can go above this one.

Reply:
`typing` is part of the standard library, so the new line should remain.